dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/StatsTaskListGenerator.java

```diff
@@ -47,7 +53,7 @@ ImmutableList<Task<?>> createTasks(ConnectorArguments arguments) throws IOExcept
   /** The source of performance statistics. */
   enum StatsSource {
     AWR("awr"),
-    METADATA("metadata"),
+    NATIVE("metadata"),
```

Comment:
I didn't mean to rename just the enum constant, but refer to the statistics that don't require installing AWR or StatsPack as `native`. The value here is still `metadata`.

Also, the directory is called `metadata`, not `native`:
https://github.com/google/dwh-migration-tools/blob/b6c687b181e3f5fb2b05e241f81fe608d54df25f/dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/OracleStatsQuery.java#L43

Reply:
Feel free to prepare the change in a separate PR.

Reply:
Done, I think we posted at the same time :)